tests/VimPlugin/project/find_project.vader

```diff
@@ -0,0 +1,28 @@
+Include (Prepare project directories): lib/create_fixtures_directory.vader
+
+Execute (Ensure that the repository is empty):
+  Assert g:phpactorProjectRepository.isEmpty()
+
+Then (Find the project in an empty repository):
+  let rootPath = fnamemodify(g:projectDirs['SillyWalksProject'], ':p')
+  let file = rootPath . '/foo.strangeext'
+
+  let project = g:phpactorProjectRepository.findProjectContainingFile(file)
+
+  AssertEqual v:null, project
```

Comment:
Should this really be null? Maybe change the test description? (e.g. `Returns null if the repository is empty`)